text/000-rustdoc-jump-to-definition.md

```diff
@@ -0,0 +1,187 @@
+Rustdoc: jump to definition
+
+- Feature Name: `jump_to_definition`
+- Start Date: 2021-12-09
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#89095](https://github.com/rust-lang/rust/issues/89095)
+
+# Summary
+[summary]: #summary
+
+Generate links on idents in the rustdoc source code pages which links to the item's definition and documentation (if available).
+
+# Motivation
+[motivation]: #motivation
+
+This RFC proposes to generate links in the rustdoc source code pages (the ones you end up on when clicking on the `source` links) to allow you to jump to an item definition or back to its documentation page.
+
+The goal of this RFC is to greatly improve the browing experience of the source page (especially when looking for private items).
+
+Since a video is worth a thousand words:
```

Comment:
Not for RFCs, RFCs should be readable. It's fine to link to a video but please please do not use a video to replace text contents, or write something that claims to do so. Let's just link to this in the guide-level explanation as a demo.

Reply:
The wording is indeed misleading. What I tried to express was: "you had the description, here you can see an example in action". Also, it should not be in the motivation section.